### DISCUSSION TITLE
Common Burn Address

### DISCUSSION THREAD

**Discussion by DEFSTALKR** [ordinals/ord#1585] [Ideas]
I've seen a few collection inscribe the same works twice, I have also done it in the beginning which is a pain, so it would be cool to be able to send the double up to a common burn address which then shows some sort of empty or "BURNED" image on the explorer. Obviously the inscription and its #ID is still valid, but allows it to not show as a duplicate on the explorer. Thoughts?

**Comment by casey** [comment]
I want to do this by sending to a new type of address, which looks like `burn1…`. We could then show `🔥 BURNED 🔥` on the /inscriptions page. Would this work? We could then decide if we wanted to hide them, de-emphasize them, or if just seeing the burned status on /inscriptions would be enough.

**Comment by DEFSTALKR** [comment]
Yes the new address would be cool. And I think simply just showing some basic burn emoji or Ord burn icon would be sufficient. Hiding them doesn’t quite make sense to me. But hiding the artwork through a burn symbol does.
